client/src/redux/sagas/datafiles.sagas.js

```diff
@@ -492,6 +492,10 @@ export async function uploadFileUtil(api, scheme, system, path, file) {
     body: formData,
   });
   if (!request.ok) {
+    if (request.status === 403 || request.status === 500) {
```

Comment:
Can we put a try catch around line 488 to line 498 to do the following? A frequent issue we see is the client side network is blocking file uploads (firewall settings), since the firewall is blocking it, sometimes, it might not even be a http response.

```
try {
    const request = await fetch(url, {
      method: 'POST',
      headers: { 'X-CSRFToken': Cookies.get('csrftoken') },
      credentials: 'same-origin',
      body: formData,
    });

    if (!request.ok) {
      throw new Error(`HTTP error: ${request.status}`);
    }

    return request;

  } catch (error) {
    if (error instanceof TypeError) {
      throw new Error('Network error: The file upload was blocked.');
    }

    throw error;
  }
  ```